docs/concepts/governance.md

```diff
@@ -41,7 +41,14 @@ The network will also calculate:
 The proposal will pass if one of the two scenarios occur: 
 1. The tokenholder vote meets or exceeds the minimum set by <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarketParam.requiredParticipation" hideValue={true} />,  and the votes in favour are greater than the amount set by <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarketParam.requiredMajority" hideValue={true} />. In this case the market's liquidity providers were overridden by governance token holders.
 2. The governance tokenholder vote does not reach participation threshold, but the liquidity providers' votes do, and there are enough votes in favour. The participation rate must be greater than/equal to <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarketParam.requiredParticipation" hideValue={true} />, and the liquidity providers' participation rate must be greater than/equal to <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarketParam.requiredParticipationLP" hideValue={true} />, and the liquidity providers' votes in favour is greater than/equal to <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarketParam.requiredMajorityLP" hideValue={true} />
-    
+
+### Proposal outcome: Successor market
+For proposals adding a new successor market, the outcome of the proposal can change even after it's been approved. 
+
+If a parent market is still in its proposed state, its successor market cannot be enacted, even if it passes the vote.
+
+Two proposals that name the same parent can be submitted and pass a governance vote. Whichever market clears its [opening auction](./trading-on-vega/trading-modes.md#auction-type-opening) first gets the share of the insurance pool, and the liquidity providers' equity-like share is moved to that market. The second market will still open for normal trading, assuming it gets out of opening auction, but it will not be considered a successor market.
```

Comment:
Second market will be cancelled I believe

Reply:
This is what the spec says, but I can check on the actual implementation if you think this is wrong. 

> if there are two proposals naming the same parent market then whichever one gets into the active state first (i.e. > passes governance vote and clears the opening auction) becomes the successor of the named parent; the other > proposal is kept but the parent market id field is cleared and when opening auction ends no virtual stake will get > carried over.

Reply:
Seems we have some spec contention as there is an AC saying:


> With two successor markets in opening auction, that have the same parent market, and one additional market in the state "Proposed". Get one of the two markets to leave the opening auction (passage of time, LP commitment, crossing trade). The other market in auction and the proposed market should both be "Rejected" and all LP funds will be released ([0081-SUCM-014](https://github.com/vegaprotocol/documentation/pull/689#0081-SUCM-014))

I guess we need to decide which version they should be doing (I don't have a strong preference either way so perhaps lean towards whatever the code does @EVODelavega )

Reply:
Yep and there's another one saying: 

Two proposals that name the same parent can be submitted. Both can be approved by governance. The proposed market that clears the opening auction first gets the share of the insurance pool and the virtual stakes get carried over. Once the first market clears the opening auction the other market's parent market Id field is cleared. When it clears the opening auction it gets no insurance pool from the parent and no virtual stakes get carried over ([0081-SUCM-005](https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/0081-SUCM-successor_markets.md#0081-SUCM-005)).